unicorn/include/cocaine/zookeeper/connection.hpp

```diff
@@ -0,0 +1,141 @@
+/*
+* 2015+ Copyright (c) Anton Matveenko <antmat@yandex-team.ru>
+* All rights reserved.
+*
+* This program is free software; you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation; either version 2 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*/
+#ifndef ZOOKEEPER_CONNECTION_HPP
+#define ZOOKEEPER_CONNECTION_HPP
+
+#include "cocaine/zookeeper/session.hpp"
+#include "cocaine/zookeeper/handler.hpp"
+
+#include <zookeeper/zookeeper.h>
+
+#include <vector>
+#include <string>
+
+namespace zookeeper {
+
+class cfg_t {
+public:
+    class endpoint_t {
+    public:
+        endpoint_t(std::string _hostname, unsigned int _port);
+
+        std::string
+        to_string() const;
+
+    private:
+        std::string hostname;
+        unsigned int port;
+    };
+
+    cfg_t(std::vector<endpoint_t> endpoints, unsigned int recv_timeout);
```

Comment:
Please describe timeout resolution (s, ms) either in a variable name or in comment.